committeeoversightapp/management/commands/load_cms_content.py

```diff
@@ -21,14 +21,19 @@ def handle(self, **options):
         initial_data_file_custom_pages = os.path.join(fixtures_dir, 'initial_cms_content_custom_pages.json')
 
         # Delete existing Wagtail models
-        Site.objects.all().delete()
-        Page.objects.all().delete()
-        PageRevision.objects.all().delete()
-        Image.objects.all().delete()
-        LandingPage.objects.all().delete()
-        StaticPage.objects.all().delete()
-        CategoryDetailPage.objects.all().delete()
-        CommitteeDetailPage.objects.all().delete()
+        try:
+            Site.objects.all().delete()
+            Page.objects.all().delete()
+            PageRevision.objects.all().delete()
+            Image.objects.all().delete()
+            LandingPage.objects.all().delete()
+            StaticPage.objects.all().delete()
+            CategoryDetailPage.objects.all().delete()
+            CommitteeDetailPage.objects.all().delete()
+            HearingListPage.objects.all().delete()
```

Comment:
Oo nice. This is deleting some but not all models in the `committeeoversightapp`, `wagtailcore`, and `wagtailimages` apps. Is there a different method you'd suggest here?

Reply:
Ah, that's a tough one -- I didn't realize it was a selection of models instead of the full set.

One approach that would be more object-oriented would be to define an optional class attribute, something like `Model.reset_on_load`, that determines whether or not the model should be cleared when loading data. Then you could set this attribute on models you control, and update the reset code to be something like:

```python
# Do this after deleting Wagtail models, since you don't have control over them
for Model in apps.get_app_config('committeeoversightapp').get_models():
    if Model.reset_on_load:
        Model.objects.all().delete()
```

However, this doesn't really reduce the maintenance burden introduced by this functionality, it simply moves it from the management command to the models themselves. I think this is probably a better place for it to be, since a developer adding a new model down the road is more likely to copy what they see in `models.py` than to notice they need to update a management command, but it's ultimately a matter of taste.